src/main/java/com/oracle/nosql/spring/data/repository/support/SimpleNosqlRepository.java

```diff
@@ -256,13 +256,13 @@ public String getDurability() {
     }
 
     static String convertDurability(Durability durability) {
-        if (durability == Durability.COMMIT_NO_SYNC) {
+        if (durability.equals(Durability.COMMIT_NO_SYNC)) {
```

Comment:
This ends up in NPE if durability is null.

Reply:
Fixed